scripts/rollup/build.js

```diff
@@ -174,6 +173,31 @@ function getBabelConfig(
   return options;
 }
 
+let getRollupInteropValue = id => {
+  // ReactNoop.js and ReactNoopPersistent.js use several default imports from other internal packages.
+  // That gets turned into a `require()`, but the `module.exports = function reconciler()` wrapper
```

Comment:
This comment seems outdated now.

Can we instead write the criteria by which we determine what goes into this list?